<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="file_path" />
  <!-- Start the node -->
  <node name="scan_node" ns="simulation_groundtruth" pkg="simulation_groundtruth" type="scan_node" output="screen">
    <param name="file_path" value="$(arg file_path)" />
    <rosparam file="$(find simulation_groundtruth)/param/scan/default.yaml" command="load" />
    <rosparam file="$(find simulation_groundtruth)/param/scan/topics.yaml" command="load" ns="topics" />
  </node>
</launch>
